Mage/src/main/java/mage/game/turn/FirstCombatDamageStep.java

```diff
@@ -1,75 +0,0 @@
-
-package mage.game.turn;
-
-import java.util.UUID;
-
-import mage.constants.PhaseStep;
-import mage.game.Game;
-import mage.game.combat.CombatGroup;
-import mage.game.events.GameEvent;
-import mage.game.events.GameEvent.EventType;
-
-/**
- * @author BetaSteward_at_googlemail.com
- */
-public class FirstCombatDamageStep extends Step {
-
-    public FirstCombatDamageStep() {
-        super(PhaseStep.FIRST_COMBAT_DAMAGE, true);
```

Comment:
Be careful, its under usage:

https://github.com/search?q=repo%3Amagefree%2Fmage%20FIRST_COMBAT_DAMAGE&type=code

Reply:
Oops, that's why I made a PR